pkg/apiserver/apiserver.go

```diff
@@ -174,7 +174,8 @@ func InstallVersionHandler(mux Mux, container *restful.Container) {
 			Doc("get the code version").
 			Operation("getCodeVersion").
 			Produces(restful.MIME_JSON).
-			Consumes(restful.MIME_JSON))
+			Consumes(restful.MIME_JSON).
+			Writes(version.Info{}))
```

Comment:
swagger-spec is auto generated from this code.
@dims You are right that the response right now is correct. But our documentation of response (in swagger spec) is incorrect. This change just fixes that.